doc/troubleshooting/index.md

```diff
@@ -2,6 +2,14 @@
 In consequence of various differences of computers or systems, problems may occur. Some common circumstances are listed as follows. 
 If other unexpected problems occur, you're welcome to contact us for help.
    
-- [installation](installation.md)
-- [md-version-compatibility](md-version-compatibility.md)
-- [model-compatability](model-compatability.md)
+
+[Installation](installation.md)
+
+[MD: cannot run LAMMPS after installing a new version of DeePMD-kit](md-version-compatibility.md)
+
+[Model compatability](model-compatability.md)
+
+[The temperature undulates violently during early stages of MD](md-energy-undulation.md)
+
+[Do we need to set rcut < half boxsize ?](rcut.md)
+
```

Comment:
I would prefer the itemized presentation.

Reply:
OK